ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/services/tracking/TrackerImplTest.scala

```diff
@@ -87,7 +84,12 @@ class TrackerImplTest
       "return a RESOURCE_EXHAUSTED error" in {
 
         sut.track(input(1))
-        whenReady(sut.track(input(2)).failed)(IsStatusException(Status.RESOURCE_EXHAUSTED))
+        whenReady(sut.track(input(2)))(failure => {
```

Comment:
I prefer the `{ failure =>` syntax but no big deal.

Reply:
Not sure what you mean. 
The `{ failure => }` syntax that was introduced in this PR or the existing way?

Reply:
Introduced here, I prefer:
```
        whenReady(sut.track(input(2))){ failure =>
```
you can ignore

Reply:
Ah ok, got it...tend to agree with you tbh 👍